README.md

```diff
@@ -199,6 +199,9 @@ if [ -f $NATALIE_PATH ]; then
     else
         echo "Natalie Generator: Generated Swift is up-to-date; skipping re-generation."
     fi
+else 
+    echo "error: Could not find Natalie Generator at $NATALIE_PATH; Please visit https://github.com/krzyzanowskim/Natalie for installation instructions."
+    exit 1
 fi
```

Comment:
ok agree with the modification

as you say we could make a warning like swiftlint https://github.com/realm/SwiftLint#xcode

but in our case, not generating the file could be more annoying (someone edit the storyboard and do not update the file because the tool is not installed, so the app will not be consistent)

so I am for the error as default proposition

Reply:
Yea, I agree